js/src/redux/selectors/fallbacks.js

```diff
@@ -9,7 +7,7 @@ const socialSelectors = selectorsFactory( "social" );
  *
  * @returns {string} The fallback title.
  */
-const getTitleFallback = state => state.analysisData.snippet.title;
+export const getTitleFallback = state => get( state, "analysisData.snippet.title", "" );
```

Comment:
These are not used.
Also, I was wondering... Why don't we let `getTwitterTitle` or `getFacebookTitle` handle all the fallbacks they should get? I don't want to have to write:
```js
title: getTwitterTitle() || getTitleFallback();
```
in the container. I feel like I can expect `getTwitterTitle()` to handle that.

Just my opinion though.

Reply:
Nor do I want to write:
```js
title: getTwitterTitle();
titleFallback: getTitleFallback();
```

Reply:
The problem is that this will also populate the input field with the data from the store. That is not what we want. Therefore, we need to have a separate selector.